smartsim/experiment.py

```diff
@@ -158,7 +157,7 @@ def __init__(self, name: str, exp_path: str | None = None):
         experiment
         """
 
-    def start(self, *jobs: Job | t.Sequence[Job]) -> tuple[LaunchedJobID, ...]:
+    def start(self, *jobs: Job | t.Sequence[Job]) -> tuple[Record, ...]:
```

Comment:
Do you need to update the :return: ?

Reply:
d'oh! 🤦 I knew I let  a few of these slip through the cracks... yeah! that should be updated!!